docs/CHANGELOG.asciidoc

```diff
@@ -44,6 +45,265 @@ Review important information about the {kib} 8.x releases.
+Maps::
+* Adds a new layer wizard for the spatial join option {kibana-pull}156618[#156618]
+Metrics::
+* Adds copilot to hosts process {kibana-pull}159413[#159413]
+Observability::
+* Adds TLS Certificates page copied from Uptime, which only displays certificates from the synthetics application {kibana-pull}159541[#159541]
+* Adds creating, editing, deleting monitors in private locations, which no longer requires all permissions in feet. You will only need synthetics write permissions. {kibana-pull}159378[#159378]
+* Adds burn rate windows to SLO detail page {kibana-pull}159750[#159750]
+* Adds *Normalize by* time and scale factor on Diff TopN functions page {kibana-pull}159394[#159394]
+* Adds a severity label to SLO burn rate rule reason message {kibana-pull}158954[#158954]
+* Adds Copy JSON button to Clipboard for SLO form {kibana-pull}157902[#157902]
+* Adds Custom Metric SLI {kibana-pull}157421[#157421]
+* Adds Data Views to index pattern selector {kibana-pull}158033[#158033]
+Platform::
+* The savedObjects export API now supports exporting all types using the `*` wildcard. Please refer to the documentation {kibana-pull}159289[#159289]
+Security::
+* Hide create spaces button when limit is reached {kibana-pull}159102[#159102]
+Uptime::
+* Adds the option to send an HTML email instead of a plaintext email to users when assigned to a case {kibana-pull}159335[#159335]
+* Implement standard time formatting {kibana-pull}143799[#143799]
+* Adds a new field, called `category`, to categorize cases. Users can create a new category, set or select an existing one on a case, and filter by multiple categories on the cases table. {kibana-pull}159890[#159890]
+
+[float]
+[[fixes-v8.9.0]]
+=== Bug Fixes
+Alerting::
+* Fixes containment boundaries not being re-fetched when a query changes {kibana-pull}157408[#157408]
+* Fixes the charts on Log threshold breached details page {kibana-pull}160321[#160321]
+APM::
+* Fixes infinite loading of APM alert table {kibana-pull}161134[#161134]
+* Fixes other bucket message and sorting {kibana-pull}159919[#159919]
+* Fixes percentiles for service transaction metrics {kibana-pull}158913[#158913]
+* Fixes stack trace on errors when only available as plain text {kibana-pull}156831[#156831]
+Cases::
+* Fixes a bug in the alerts table where you cannot create a new case when attaching alerts to a case from the cases modal {kibana-pull}160526[#160526]
+* Fixes an issue where the following special characters could not be included in the case tags: `\\():<>"*` because it resulted in a bug where the case would not be displayed in the cases table when filtered for those tags. These characters are now handled correctly and the cases will be shown in the table. {kibana-pull}159815[#159815]
+Dashboard::
+* Fixes an issue where it was possible for a URL update (redirecting to the alias) to happen in the middle of building the errored state of the Dashboard {kibana-pull}161043[#161043]
```

Comment:
I would reword this to something like:

`Fixes an issue where a Dashboard alias redirect would show a blank screen instead of redirecting properly.`

Reply:
Applied change